imscp.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+
+from lxml import etree
+import os
+import re
+import shutil
+import tempfile
+import zipfile
+
+from ricecooker.classes import nodes, files, licenses
+from ricecooker.utils.zip import create_predictable_zip
+from ricecooker.utils.browser import preview_in_browser
+
+
+def extract_from_zip(zip_file_path, license):
+    with tempfile.TemporaryDirectory() as extract_path:
+        print('Extracting zip file %s to %s' % (zip_file_path, extract_path))
```

Comment:
Minor nitpick, but could we switch these lines to use the `logging` module instead since this will ultimately be used as a library? `__main__` can call `logging.basicConfig` and set it up to print to the console so that the behavior is the same.